Automattic/sensei
# Fix Sensei notice styling and button output

Resolves #6692

## Proposed Changes
* Vertically centres notice text on non-Sensei home page notices, and adjusts colouring, border, etc., to match
* Changes `{ action.label }` to be output with `<RawHTML></RawHTML>` similar to how `{ messages }` is already output in this way.

## Testing Instructions

1. Check out branch
2. Add the snippet provided in the issue report
3. Look at the fixed up notices

![sensei-courses](https://user-images.githubusercontent.com/2500940/228528801-6f593f2f-fe77-41b5-a6b4-94eb39681f8d.png)
![sensei-home](https://user-images.githubusercontent.com/2500940/228528803-126677ea-773d-4daf-ac41-1a57e5cb762f.png)

## Pre-Merge Checklist
